frontend/src/components/whaleopedia-page/WhaleCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I don't think you use either of these:
```suggestion
import React from "react";
```